### DISCUSSION TITLE
addIterativeLSI - Definition of terms

### DISCUSSION THREAD

**Discussion by Dazcam** [GreenleafLab/ArchR#1288] [Questions / Documentation] [answered]
Hello,

Could you advise on the precise definition of the following `clusterParams` terms in the `addIterativeLSI` function:

1. `n.start`
2. `maxClusters`

This is not clear from the function documentation [here](https://www.archrproject.com/reference/addIterativeLSI.html) or when running `?addIterativeLSI` in R.

For context, I'm looking to confirm the parameter I should alter for:

> (1) reducing the maximum number of clusters identified in the first iteration 

This is given as the first option to try when looking to attenuate strong batch effects ([page 27](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-021-00790-6/MediaObjects/41588_2021_790_MOESM1_ESM.pdf)), but, as it stands, a clear explanation for what both these parameters do is not provided.

Many Thanks.

**Comment by rcorces** [comment, marked as answer]
`maxClusters` is a parameter passed to `ArchR::addClusters()`
https://www.archrproject.com/reference/addClusters.html

`n.start` is a parameter passed to `Seurat::FindClusters()`
https://satijalab.org/seurat/reference/findclusters

**Reply by Dazcam** [reply]
Many thanks.
